web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/UtilsApiController.java

```diff
@@ -185,10 +186,10 @@
         return new ResponseEntity<>(sb.toString(), HttpStatus.OK);
     }
 
-    private String getCancerType(OncoTreeType oncoTreeType) {
+    private String getCancerType(TumorType oncoTreeType) {
         return oncoTreeType == null ? null : (
-            oncoTreeType.getSubtype() == null ?
-                oncoTreeType.getCancerType() :
-                oncoTreeType.getSubtype());
+            oncoTreeType.getName() == null ?
```

Comment:
Ideally, both properties shouldn't be empty, but since here we are returning the name, so we should check the name not the code.